cmd/harmony/main.go

```diff
@@ -412,14 +412,17 @@ func main() {
 	if *isExplorer {
 		startMsg = "==== New Explorer Node ===="
 	}
-	utils.GetLogInstance().Info(startMsg,
-		"BlsPubKey", hex.EncodeToString(nodeConfig.ConsensusPubKey.Serialize()),
-		"ShardID", nodeConfig.ShardID,
-		"ShardGroupID", nodeConfig.GetShardGroupID(),
-		"BeaconGroupID", nodeConfig.GetBeaconGroupID(),
-		"ClientGroupID", nodeConfig.GetClientGroupID(),
-		"Role", currentNode.NodeConfig.Role(),
-		"multiaddress", fmt.Sprintf("/ip4/%s/tcp/%s/p2p/%s",
+	utils.Logger().
+	Info().
+	Str("", startMsg).
```

Comment:
`startMsg` is the message string.  Move this to the end of the event chain as `Msg(startMsg)`.